### PR TITLE
Laser pointers. Now more accessible!

### PR DESCRIPTION
**Pull request by dis-integrates-the-integration-tests**
## About The Pull Request

This PR _should_ add the laser pointer (With its variations) to the loadout and the Good Clean Fun vendor.

## How This Contributes To The Skyrat Roleplay Experience

I think laser pointers count as toys, so it's only fair if people can get their hands on them more often!
See someone with felinid traits? Someone that looks a little cat-ty..?
**LIGHT!**
Or.. you just want to be generally annoying to that one friend?
**LIGHT!**
Or.. perhaps.. You wanna get someone's attention without pointing like some maniac?
**LIGHT!**
Or or- Maybe- You just wanna be that one annoying bastard to borgs?
**LIGHT!**

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

  
image
image
image

</details>

## Changelog

:cl:
qol: Laser pointers are now more accessible, being now available in the loadout and in the Good Clean Fun vendor.
/:cl:

### CODE REVIEW

**Comment by HoodMilk** [comment]
People cannot resist laserpointering felinids when they're in the middle of roleplaying, even with only 2 laser pointers on the station, it's very lame. If you really want a laser pointer you can just buy them from the cargo crate or ask a miner. I feel like the kind of person taking this in their loadout is just going to be annoying with it.

**Comment by Ephemeralis** [comment]
For full disclosure, laser pointers are far more powerful than they readily appear. A t3+ laser pointer is capable of remotely flashing/stunning cyborgs, flashing/blinding organics and EMPing cameras for 40+ seconds from 7 tiles away, *potentially through walls* when a bluespace crystal is added to them past t3 as well.

Part of me wants to argue that they shouldn't be common, but they're already readily available via bureaucracy crates from cargo and the RD + lawyer start with them.

We likely need to address the felinid forced movement stuff in a separate PR. I think it's one of those lingering 'felinid are a joke race so we must grief them at every opportunity' things that's still held over from tgstation upstream.

File: code/modules/vending/games.dm [hunk 1]
```diff
@@ -25,6 +25,9 @@
 			"products" = list(
 				/obj/item/toy/captainsaid = 1,
 				/obj/item/toy/intento = 3,
+				// SKYRAT EDIT CHANGE - Adds laser pointers to the Good Clean Fun vendor
+				/obj/item/laser_pointer = 3,
```
Comment (theselfish) [inline code/modules/vending/games.dm]: Not the way to do this. Look in the modular_vending file for the way we do this.

**Comment by theselfish** [comment]
If we're willing to add medkits, toolboxes and other such items to the loadout. I don't see why this should be much different.

File: code/modules/vending/games.dm (hunk 1 above)
Comment (dis-integrates-the-integration-tests) [reply code/modules/vending/games.dm]: Ah
I'll look into it when I'm on laptop

Comment (dis-integrates-the-integration-tests) [reply code/modules/vending/games.dm]: ok, fixed

**Comment by dis-integrates-the-integration-tests** [comment]
> For full disclosure, laser pointers are far more powerful than they readily appear. A t3+ laser pointer is capable of remotely flashing/stunning cyborgs, flashing/blinding organics and EMPing cameras for 40+ seconds from 7 tiles away, _potentially through walls_ when a bluespace crystal is added to them past t3 as well.
> 
> Part of me wants to argue that they shouldn't be common, but they're already readily available via bureaucracy crates from cargo and the RD + lawyer start with them.
> 
> We likely need to address the felinid forced movement stuff in a separate PR. I think it's one of those lingering 'felinid are a joke race so we must grief them at every opportunity' things that's still held over from tgstation upstream.

We could arrange something!  I have nothing against that if people see it necessary.

**Review by GoldenAlpharex — APPROVED** [review]
I'm okay with this, however I'm going to request that you try to be less vague with your changelogs in the future. If something's more accessible, mention _where_ you can now get it from. I'll modify your changelog for you this time around.

Further modifications to laser pointers can be made in a separate PR, so they can be properly reviewed, yeah.

**Comment by dis-integrates-the-integration-tests** [comment]
> I'm okay with this, however I'm going to request that you try to be less vague with your changelogs in the future. If something's more accessible, mention _where_ you can now get it from. I'll modify your changelog for you this time around.
> 
> Further modifications to laser pointers can be made in a separate PR, so they can be properly reviewed, yeah.

Understood!
I thought the CL would be a little too long. I'll keep it in mind (insert thumbs up)

**Comment by Nerev4r** [comment]
image
All RP involving borgs, felinids, a camera, or anyone without sunglasses is REALLY going to take a hit now. There's not even any indication as to who is using a laser pointer at any given time. We really need to adjust this in a future PR.


I'd suggest adding a Beam() between the user and the pointer at the least.

**Pull request merged**